### DISCUSSION TITLE
Vote of Thanks to all Contributors 💘

### DISCUSSION THREAD

**Discussion by pooranjoyb** [pooranjoyb/popShop#396] [Announcements]
# Thank You So Much💘 

_**Thanks to all the Contributors who gave their invaluable love and support to **PopShop**!**_
This 3 months had been an amazing experience with you guys. Loved to see your spirit and enthusiasm towards the development of this amazing eCommerce portal.

### Our Achievements 🚀 
- We completed a Full-Stack end-to-end working eCommerce Platform from scratch.
- Also deployed the entire portal in production as well as dev.
- We integrated almost all eCommerce functionalities, such as
    - User oauth authentication via Supabase Auth 
    - Managing Orders + Cart functionalities.
    - Checkout and Payment Management
    - Admin Dashboard to track user-onboarding
    - Admin Portal to Upload Products
    - User Data Management with Admin Portal
    - Orders Management from Admin Portal
- We excelled in TypeScript Tech Stack through this project. 
- Learning and Exploration of BaaS with Supabase
- Learning State Management with Redux-Toolkit.
- We created the complete UI Daisy UI + Tailiwind.
- Successfully setup the CI/CD pipeline with Github actions.
- Documenting most of the Codebase wherever necessary.

### **_Would be glad if you guys vote us as your favorite PA and Mentors in the official GSSoC'24 Form circulated in your Email or you can just click the links below :)_**

- **Link for choose your favorite PA : [Click here](https://docs.google.com/forms/d/e/1FAIpQLSfp1TBUXoAY8BN9Y0H0UYUP_YSI75Mz8UWSt9HbR0PZOwQwmw/viewform)**
- **Link to choose your favorite Mentor : [Click here](https://docs.google.com/forms/d/e/1FAIpQLSddEocxJH71cidNDvOwtYYvFwE4uz2OIxWtLy7SD4Eimz_R9A/viewform)**

### Contributors :
@dhruv8433 @abckhush @kapilG0 @SandipGyawali @gdhyani @Disha-Hati @Mujtabaa07 @NIveX2169 @masterboy376 @UtkarshBirla28 @Asymtode712 @sanskritisahu21 @Tanishashrivas @bhuvangoel04 @BHS-Harish @Sid-80 @sristy17 @yashi-15 @AkhilaSunesh @vansh-sardana @amanjaiswal7236 @Mayur-666 @Niharika0104 @dakshsinghrathore @richab246 @gdhyani @yash28goyal @me-shweta @aryansharma220 @Sourabh782 @gauravRajputPearl @RamakrushnaBiswal @yashmandi @Yogesh-ok55 @Utsavladia @somuwashere @SamIsTheFBI @nagalakshmi08 @PradnyaGaitonde @AnitSarkar123 @harini-1597 @IP80808080 @dev04sa @sachin0986 @Subham-u @jayanththalla @kapilG0 @SouptikTaran @Meetjain1 @ParnaRoyChowdhury777 @Amitooo1

**Comment by bhuvangoel04** [comment]
Thank you so much for your kind words and support! I'm truly grateful for your active involvement and the guidance you've provided throughout my journey with popShop contribution. Your insights and prompt feedback have been invaluable in helping me upskill and grow as a contributor.

**Reply by pooranjoyb** [reply]
Most welcome @bhuvangoel04 🚀 
Keep your open source spirit high! ⚡

**Comment by RamakrushnaBiswal** [comment]
Great experience.working with you ⭐ @pooranjoyb 💥

**Reply by pooranjoyb** [reply]
Thanks @RamakrushnaBiswal ❤️

**Comment by masterboy376** [comment]
Thank you so much @pooranjoyb for your recognition. It was a great experience working with all. 👏💐

**Reply by pooranjoyb** [reply]
Haha, Means a lot @masterboy376 🚀

**Comment by Meetjain1** [comment]
It was great contributing in this project @pooranjoyb .

**Reply by pooranjoyb** [reply]
Thanks @Meetjain1 🚀

**Comment by abckhush** [comment]
Thank you :) You have been very fun to work with and your project was very interesting as well.

**Reply by pooranjoyb** [reply]
Haha ikr, thanks to you for your invaluable contributions :)

**Comment by me-shweta** [comment]
Hey @pooranjoyb ! It was my first time contributing to open source, but you motivated me and helped me understand it. Thank you :)

**Reply by pooranjoyb** [reply]
Most welcome @me-shweta Keep contributing, Keep shining 🚀
